core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -115,7 +115,7 @@ class AnalyzerManager(
     case req @ CompletionsReq(fileInfo, _, _, _, _) =>
       getOrSpawnNew(config.findProject(fileInfo)) forward req
     case req @ UsesOfSymbolAtPointReq(file, _) =>
-        getOrSpawnNew(config.findProject(file)) forward req
+      getOrSpawnNew(config.findProject(file)) forward req
     case req @ SymbolAtPointReq(file, point: Int) =>
       getOrSpawnNew(config.findProject(file)) forward req
```

Comment:
this pattern is repeating a lot. I wonder if perhaps it would be cleaner to have a method (typeclass) that extracts the `SourceFileInfo` / `Option[EnsimeProjectId]` then we don't need to repeat the business logic.